usbh_midi.cpp

```diff
@@ -147,10 +147,9 @@ uint8_t USBH_MIDI::Init(uint8_t parent, uint8_t port, bool lowspeed)
         p->epinfo = epInfo;
         p->lowspeed = lowspeed;
 
-        // Get device descriptor
-        rcode = pUsb->getDevDescr( 0, 0, sizeof(USB_DEVICE_DESCRIPTOR), (uint8_t*)buf );
-        vid = udd->idVendor;
-        pid = udd->idProduct;
+        // First Device Descriptor Request (Initially first 8 bytes)
```

Comment:
Could leave a link to this page as a comment: https://techcommunity.microsoft.com/t5/microsoft-usb-blog/how-does-usb-stack-enumerate-a-device/ba-p/270685?

Reply:
I'll do